recipe/meta.yaml

```diff
@@ -41,12 +40,13 @@ outputs:
       requires:
         - pip
 
-  - name: py-rattler
+  - name: py_rattler
     build:
       noarch: generic
+      skip: true  # [not linux64]
     requirements:
       run:
-        - {{ pin_subpackage('py_rattler', max_pin="x.x.x") }}
```

Comment:
should we use `exact=true` here?

Reply:
https://github.com/conda-forge/py-rattler-feedstock/pull/9#discussion_r1559024100